### PR TITLE
Update pjc lib

### PR DESCRIPTION
**Pull request by camallen**
_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

Package: fe-project, lib-classifier

This PR updates the underlying PJC to v3.3.2, specifically to include the changes that improve the bearer token refresh event handling and decrease the time expose to scenarios where the API token may have expired.

See https://github.com/zooniverse/panoptes-javascript-client/blob/master/CHANGELOG.md#v332-2021-05-07 for specific version changes

~Also this PR removes the PJC as a `devDependency` in lib-classifier. I'm not sure why this is included as the fe-project injects it's version of the PJC via the classifierWrapper. If this is for testing perhaps better use of test doubles here can help vs the `devDependency` - TBC pending the test runs.~

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Review by srallen — CHANGES_REQUESTED** [review]
The dev dependency in the classifier library is used for an isolated SPA we use development and manual testing purposes. It's found here: https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/dev

And can be run by `yarn dev` when in the library folder. Could you please restore it?

**Review by eatyourgreens — APPROVED** [review]
Looks good to me. The monorepo bootstraps, and I can run the dev classifier. I can submit classifications from I Fancy Cats, using the updated client.

Maybe add PJC 3.3.2 as a dev dependency of the classifier, so that it can build outside the context of the monorepo. At the moment, the dev classifier hoists the client from the project app.

File: packages/app-project/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "newrelic": "~7.1.0",
     "next": "~10.1.3",
     "node-fetch": "~2.6.1",
-    "panoptes-client": "~3.2.1",
+    "panoptes-client": "3.3.2",
```
Comment (eatyourgreens) [inline packages/app-project/package.json:55]:
```suggestion
    "panoptes-client": "~3.3.2",
```

File: packages/lib-classifier/package.json [hunk 2]
```diff
@@ -87,7 +87,6 @@
     "mocha": "~8.3.2",
     "mst-middlewares": "~3.17.0",
     "nock": "~13.0.11",
-    "panoptes-client": "~3.0.0-rc.0",
     "polished": "~3.6.4",
```
Comment (eatyourgreens) [inline packages/lib-classifier/package.json:90]:
```suggestion
    "panoptes-client": "~3.3.2",
    "polished": "~3.6.4",
```

File: packages/app-project/package.json (hunk 1 above)
Comment (camallen) [reply packages/app-project/package.json:55]: i noticed this - i found it a little strange that `yarn upgrade` didn't respect the ~ already there.

Comment (srallen) [reply packages/app-project/package.json]: Using `yarn upgrade <package>@<version>` will upgrade the package to the exact version you specify (which is different behavior from npm). Use `yarn upgrade <package> --latest` for it to follow the existing range specifier. I prefer using `yarn upgrade-interactive --latest`.

**Review by srallen — APPROVED** [review]

**Pull request merged**